### PR TITLE
Correct a few things and make branding consistent with overte

### PR DESCRIPTION
**Pull request by akamicah**
- Updated `metaverseConfig`to point to the correct url for the overte metaverse server
- Removed unnecessary commented out code
- Changed Vircadia footer branding to Overte, with a link to the Overte contact page
- Removed Kalila avatar pop-up from home page.

### CODE REVIEW

File: src/store/index.js [hunk 1]
```diff
@@ -3,6 +3,7 @@
 //
 //  Created by Kalila L. on 2 Jul 2020.
 //  Copyright 2020 Vircadia contributors.
+//  Modified for Overte. 2022 Overte e.V.
```
Comment (JulianGro) [inline src/store/index.js]: The way we have always handled this is to add our copyright:
```suggestion
//  Copyright 2022 Overte e.V.
```

File: src/views/Home.vue [hunk 2]
```diff
@@ -40,32 +40,6 @@
...
-                                    large
-                                    target="_blank"
-                                    v-on="on"
-                                >
-                                    <v-icon large>mdi-heart</v-icon>
-                                </v-btn>
-                        </template>
-                        <span>
-                            <v-avatar color="pink" size="72" class="mr-2">
-                                <v-img src="assets/231352681.png"></v-img>
```
Comment (JulianGro) [inline src/views/Home.vue:62]: Please remove 231352681.png as well, unless it is still used somewhere.

File: src/store/index.js (hunk 1 above)
Comment (akamicah) [reply src/store/index.js]: Done

File: src/views/Home.vue (hunk 2 above)
Comment (akamicah) [reply src/views/Home.vue:62]: Done, also removed an additional unnecessary image

**Review by JulianGro — APPROVED** [review]

**Comment by Colorfingers** [comment]
I was able to  run the server and test out the changes.  It looks real clean.  Great work Micah!  It get's my purple badge of approval!

**Pull request merged**